Packs/MandiantAdvantageAttackSurfaceManagement/Integrations/AttackSurfaceManagement/AttackSurfaceManagement.py

```diff
@@ -0,0 +1,520 @@
+            raise DemistoException('The ASM API was unable to return details for'
+                                   f'issue {issue_id} in project {self.project_id}')
+
+        return response['result']
+
+    def get_notes(self,
+                  resource_type: str,
+                  resource_id: str) -> dict:
+        endpoint = f'notes/{resource_type}/{resource_id}'
+
+        response = self.make_request('GET', endpoint,
+                                     headers={'PROJECT_ID': str(self.project_id)})
+
+        if not isinstance(response, dict) or not response.get('success'):
+            raise DemistoException('The ASM API was unable to return notes for'
+                                   f'{resource_type} {resource_id} in project {self.project_id}')
+
+        return response['result']
+
+    # TODO: REMOVE the following dummy function:
+    def baseintegration_dummy(self, dummy: str) -> Dict[str, str]:
+        """Returns a simple python dict with the information provided
+        in the input (dummy).
+
+        :type dummy: ``str``
+        :param dummy: string to add in the dummy dict that is returned
+
+        :return: dict as {"dummy": dummy}
+        :rtype: ``str``
+        """
+
+        return {"dummy": dummy}
+    # TODO: ADD HERE THE FUNCTIONS TO INTERACT WITH YOUR PRODUCT API
+
+
+''' HELPER FUNCTIONS '''
+
+# TODO: ADD HERE ANY HELPER FUNCTION YOU MIGHT NEED (if any)
+
+
+def helper_create_incident(issue: dict, project_id: int) -> dict:
+    severity_map = {1: 4, 2: 3, 3: 2, 4: 1, 5: 0.5}
+
+    raw_json = issue
+    raw_json['mirror_direction'] = 'Both'
+    raw_json['mirror_instance'] = demisto.integrationInstance()
+
+    issue = {
+        'name': issue['summary']['pretty_name'],
+        'details': json.dumps(issue, indent=4),
+        'occurred': parser.parse(issue['first_seen']).isoformat(),
+        'severity': severity_map.get(issue['summary']['severity']),
+        'rawJSON': json.dumps(raw_json),
+        'dbotMirrorId': issue.get('id', issue.get('uid')),
+        'CustomFields': {
+            'masmcollection': issue['collection'],
+            'masmproject': str(project_id),
+            'masmconfirmed': issue['summary']['confidence'] == 'confirmed',
+            'masmentityname': issue['entity_name'],
+            'masmcategory': issue['summary']['category']
+        }
+    }
+
+    return issue
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        client.get_projects_list()
+
+        fetch_incidents(client)
```

Comment:
i would suggest to test it only if `fetch_incidents` has been marked.

```suggestion
        if demisto.params().get('isFetch'):
            fetch_incidents(client)
```

something like this, that indicates that fetch is marked and the user wants to use the fetch.